tox.ini

```diff
@@ -12,4 +12,4 @@ deps =
 
 commands =
     coverage erase
-    coverage run --branch --source {toxinidir}/auto_nag {envbindir}/nosetests -v --rednose --force-color {toxinidir}/auto_nag
+    coverage run --branch --source {toxinidir}/auto_nag {envbindir}/nosetests -v --rednose --nocapture --force-color {toxinidir}/auto_nag
```

Comment:
please make this change in a separate commits (or better, PR).
Now, this change in a commit called "Minor fixes".